gfortran_utils.sh

```diff
@@ -93,10 +93,34 @@ if [ "$(uname)" == "Darwin" ]; then
     GFORTRAN_DMG="${GF_UTIL_DIR}/archives/gfortran-4.9.0-Mavericks.dmg"
     GFORTRAN_SHA="$(shasum $GFORTRAN_DMG)"
 
+    function install_arm64_cross_gfortran {
+        curl -L -O https://github.com/isuruf/gcc/releases/download/gcc-10-arm-20210112/gfortran-darwin-arm64.tar.gz
```

Comment:
Right - but just to check - do you think we can depend on this curl command working for the next few years?

Reply:
Sure.

Reply:
We can also make a release in `MacPython/gfortran-install` and attach it there.

Reply:
Yes, maybe that would be the best way.